package.json

```diff
@@ -20,12 +30,42 @@
   },
   "dependencies": {
     "babel-types": "^7.0.0-alpha.15",
-    "babylon": "^7.0.0-alpha.16",
+    "babylon": "^7.0.0-beta.18",
     "chalk": "^2.0.1",
-    "commander": "^2.11.0"
+    "commander": "^2.11.0",
+    "ejs": "^2.5.7",
+    "purecss": "^1.0.0"
   },
   "devDependencies": {
-    "jest": "^20.0.4"
+    "babel-core": "6.25.0",
+    "babel-loader": "7.1.1",
+    "babel-plugin-transform-react-jsx": "6.24.1",
+    "babel-preset-env": "1.6.0",
+    "babili-webpack-plugin": "0.1.2",
+    "classnames": "2.2.5",
+    "css-loader": "0.28.4",
+    "exports-loader": "0.6.4",
+    "extract-text-webpack-plugin": "3.0.0",
+    "jest": "^20.0.4",
+    "mkdirp": "0.5.1",
```

Comment:
`ejs` does not create a folder if it does not exist, it only creates or replaces a file. Since we will be creating a `report` in the desired folder specified by the user, we will have to make sure the folder exists.